skfp/fingerprints/atom_pair.py

```diff
@@ -63,6 +64,11 @@ class AtomPairFingerprint(FingerprintTransformer):
     count : bool, default=False
         Whether to return binary (bit) features, or their counts.
 
+    normalize: bool, default=False
+        Wheather to scale count fingerprint by the heavy atom count (HAC) to
+        obtain a proportionality to molecular size. Vector values are
+        expressed in percent and rounded to the nearest integer.
```

Comment:
Maybe "Values are expressed as percentages, rounded to the nearest integer"?

Reply:
Also, add reference to the paper that mentioned HAC scaling here, I think it's already in references list

Reply:
Agreed on plain `[2]` at the end of the description